source/_redirects.htaccess

```diff
@@ -835,4 +835,24 @@ Redirect 301 /Classroom/Troubleshooting/Delivery_Issues/testing_your_connectivit
 Redirect 301 /Classroom/Basics/Email_Infrastructure/smtp_ports.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
 Redirect 301 /Classroom/Basics/Email_Infrastructure/what_is_smtp.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
 Redirect 301 /Classroom/Basics/Email_Infrastructure/recommended_smtp_settings.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
-Redirect 301 /Classroom/Send/How_Emails_Are_Sent/smtp_overview.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
\ No newline at end of file
+Redirect 301 /Classroom/Send/How_Emails_Are_Sent/smtp_overview.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
+
```

Comment:
you added all of the redirects for subusers, but I don't see anything gone for the username/password stuff. I would check the IA, and also do a site search to make sure we didn't miss anything in the IA.

Reply:
+ 1 any for 2FA?